draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2116,7 +2116,281 @@ uCDN-A                         dCDN-B                         dCDN-C
 	<c>RFCthis</c>
 	</texttable>
 	</section>
-	
+
+	<section title="ObjectList" anchor="object-list"><t>
+		ObjectList is a metadata object describing lists of objects that can be used in the context of CI/T v2 trigger spec, trigger status resources and other contexts as required. The ObjectList object can either embed the lists of objects or point to external URL(s) that hold such lists.  ObjectList allows specification of an object list type, providing instructions on the interpretation of the object list format.</t>  
+
+	<t>
+		ObjectLists MAY be recursive, i.e. including references to secondary manifests, including references to HLS, MPEG-DASH or HSS manifests as well as additional JSON-encoded ObjectLists , etc. The party consuming the object list MUST parse all recursions based on the object list type property. When doing so, the consuming party should also detect potential loops when descendant  ObjectList points back to parent  ObjectList.
```

Comment:
1. This specification refers to the Microsoft Smooth Streaming format at MSS rather than HSS.  I don't think I care one way or another how we refer to it, but the document needs to be consistent.
2. Nit: There are some extra spaces (a) between ObjectLists and the comma, and (b) before the other 2 instances of ObjectList.